src/sentry/static/sentry/app/views/incidents/list/index.tsx

```diff
@@ -151,6 +158,15 @@ class IncidentsListContainer extends React.Component<Props> {
             </PageHeading>
 
             <Actions>
+              <Button
+                onClick={this.handleNavigateToSettings}
+                href={`/settings/${orgId}/`}
```

Comment:
@billyvg I am a bit confused how there is an `href` on the button that has a different URL than the URL you get sent to by the `onClick` callback.

Reply:
Yeah this might be better to not have a URL, I wanted to have a fallback URL in case the user wanted to interact with it as if it were a real link.